docs/security/user-certs.md

```diff
@@ -10,13 +10,23 @@ Getting a User Certificate
 --------------------------
 
 This section describes how to get and set up a personal certificate to use on OSG.
-Currently you can get a certificate from either the OSG CA or from CILogon.
-After May 31st, you will only be able to get a certificate issued by CILogon.
-Other CAs may be used; if your virtual organization (VO) requires that you get a certificate from a different CA, [contact your VO Support Center](http://www.opensciencegrid.org/?pid=1000187) for instructions.
+You need an user certificate if you are going to interact directly with OSG resources or infrastructure. 
+Examples of this would be tasks like managing OASIS, directly running jobs on OSG resources, interactingly directly with 
+OSG storage elements, or updating or querying OSG site information (such as site topology).
```

Comment:
You don't need a cert to update/query OSG site info. Only to access private contact information (e.g. email addresses/phone numbers/DNs)

Reply:
updated